### DISCUSSION TITLE
Validation Loss

### DISCUSSION THREAD

**Discussion by withkelvin** [ourownstory/neural_prophet#1252] [Q&A - get help using NeuralProphet] [answered]
My loss function is loss_func="MSE". When I want to look at the metrics after fitting, I can't seem to find for Loss_val. Am i supposed to look at MSELoss_val instead?

**Comment by LeonieFreisinger** [comment, marked as answer]
@karl-richter Can you support on this question?
